src/main/kotlin/io/snyk/plugin/ui/toolwindow/SnykToolWindowPanel.kt

```diff
@@ -443,6 +449,29 @@ class SnykToolWindowPanel(val project: Project) : JPanel(), Disposable {
                             .build()
                     )
                 }
+                is ContainerImageTreeNode -> {
+                    val issuesForImage = node.userObject as ContainerIssuesForImage
+                    val scrollPane = wrapWithScrollPane(
+                        BaseImageRemediationDetailPanel(project, issuesForImage)
+                    )
+                    descriptionPanel.add(scrollPane, BorderLayout.CENTER)
+
+                    val targetImage = project.service<KubernetesImageCache>()
+                        .getKubernetesWorkloadImages()
+                        .find { it.image == issuesForImage.imageName }
+                    val psiFile = targetImage?.psiFile
+                    val line = targetImage?.lineNumber?.let { it - 1 } // to 1-based count used in the editor
+                    if (psiFile != null && psiFile.isValid && line != null) {
+                        val document = PsiDocumentManager.getInstance(project).getDocument(psiFile)
+                        if (document != null) {
+                            val lineNumber = if (0 <= line && line < document.lineCount) line else 0
+                            val lineStartOffset = document.getLineStartOffset(lineNumber)
+                            navigateToSource(psiFile.virtualFile, lineStartOffset)
+                        }
+                    }
+                    // TODO: Add image click event logging ?
```

Comment:
Let's keep questions/potentials in Slack/Jira.

Reply:
would like to keep it close to the "source" in all senses :)
it will help me(others) later to spot the place when implementing event tracking